themes/digital.gov/layouts/shortcodes/card-quote.html

```diff
@@ -1,11 +1,26 @@
-{{- if (.Get "text") -}}
-  <div class="card-quote {{ if (.Get "bg") -}}{{- .Get "bg" -}}{{- end -}}">
+{{/* card-quote short code to display a block of quoted text with optional citation
+
+  @param {string} bg - optional value of "dark" to display a dark background color otherwise will display as gold
+  @param {string} cite - optional string to display a citation
+  @param {string} text - required text for displaying a quote
+
+  @example usage
+  {{< card-quote text="The big idea goes here" cite="Dr. Meredith Larson, Department of Education"
+>}}
+
+*/}}
+{{ $bg := (.Get "bg") }}
+{{ $cite := (.Get "cite") }}
+{{ $text := (.Get "text") }}
+
+{{- if $text -}}
+  <div class="card-quote {{ if $bg }}card-quote--{{ $bg }}{{ end }}">
     <blockquote>
-      <span class="q">“</span>{{- .Get "text" | markdownify -}}<span class="q"
-        >”</span
-      >
-      {{- if (.Get "cite") -}}
-        <cite>— {{- .Get "cite" | markdownify -}}</cite>
+      <span class="card_quote__quotation-mark">“</span>
```

Comment:
### Improvement

I've renamed `q` to prefer BEM naming convention and for clarity.